search-app/src/cmr/search/api/routes.clj

```diff
@@ -369,11 +384,11 @@
 
         ;; add routes for managing jobs
         (common-routes/job-api-routes
-          (routes
-            (POST "/refresh-collection-metadata-cache" {:keys [headers params request-context]}
-              (acl/verify-ingest-management-permission request-context :update)
-              (metadata-cache/refresh-cache request-context)
-              {:status 200})))
+         (routes
```

Comment:
Not intentional. :) And I think github should be smart enough to ignore whitespaces.